cassiopeia/solver/SpectralNeighborJoiningSolver.py

```diff
@@ -0,0 +1,318 @@
+"""
+This file stores a subclass of DistanceSolver, SpectralNeighborJoining. This 
+algorithm is based on the one developed by Jaffe, et al. (2021) in their paper 
+titled "Spectral neighbor joining for reconstruction of latent tree models",
+published in the SIAM Journal for Math and Data Science. 
+ 
+"""
+
+import itertools
+import networkx as nx
+import numpy as np
+import pandas as pd
+from scipy.linalg import svd
+from typing import Callable, Dict, List, Optional, Tuple
+
+from cassiopeia.solver.DistanceSolver import DistanceSolver, DistanceSolverError
+
+from cassiopeia.data import CassiopeiaTree
+from cassiopeia.solver import (
+    dissimilarity_functions,
+    solver_utilities,
+)
```

Comment:
These import should also be alphabetized, right?

Reply:
do you mean I should put these with the rest of the imports? or that DistanceSolver should go before CassiopeiaTree? (In DistanceSolver the CassiopeiaTree import is listed separately from the rest of the imports so I just followed that)

Reply:
Yeah, I think the best practice would put the DistanceSolver imports with the rest of the cassiopeia imports. Can you change this in `DistanceSolver.py` as well?